### PR TITLE
Refresh automotive page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Refresh automotive page

## QA

- Go to [https://ubuntu-com-12039.demos.haus/automotive](https://ubuntu-com-12039.demos.haus/automotive)
- Compare page against [copy doc](https://docs.google.com/document/d/10jSLdwz2Ks7gnEMXlKWzrpsY6TQFxISUgnQo4y3yOrA/edit#)

## Issue / Card

Fixes [#5934](https://github.com/canonical/web-squad/issues/5934)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @minkyngkm, can we update a few things please:

- Can you make the images in the strips bigger please, the [current page](https://ubuntu.com/automotive) is a good guide. Compare "autonomous cars" for example
- Make the first logo strip --light please
- Add a border hr to the top of the final logo strip

I guess we will need the final hero image before this goes live.

Thanks

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves for reviewing! Yep, I'm waiting for the final hero image. Other than that, I made some updates based on your suggestions. Hopefully it looks better now! :)

**Comment by minkyngkm** [comment]
Hi @spencerbygraves! As demo is not working, could I give you a screenshot of the page?

Boost your automotive business with Ubuntu | Ubuntu (2022-09-22 08-45-29)

**Comment by spencerbygraves** [comment]
Thanks for updating @minkyngkm, looks good.

File: templates/automotive/index.html
Comment (petesfrench) [inline templates/automotive/index.html:392]:
```suggestion
      <p>Complete control. No delayed patches, no partial updates, no unauthorised ECU software.</p>
```

File: templates/automotive/index.html
Comment (petesfrench) [inline templates/automotive/index.html:637]:
```suggestion
    <p><a href="/automotive/developers">Read more about the Ubuntu developer experience&nbsp;&rsaquo;</a></p>
```
Will this /automative/developers be live before this is merged, if not we should remove this link for now

File: templates/automotive/index.html
Comment (petesfrench) [inline templates/automotive/index.html:660]:
```suggestion
          <h4 class="p-heading-icon__title">Whitepaper</h4>
```

File: templates/automotive/index.html
Comment (petesfrench) [inline templates/automotive/index.html:703]: You might want to get another opinion on this, but to me this feels a little off-brand.

File: templates/automotive/index.html
Comment (petesfrench) [inline templates/automotive/index.html:53]:
```suggestion
        hi_def=True,
        loading=lazy,
```

File: templates/automotive/index.html
Comment (petesfrench) [inline templates/automotive/index.html:545]:
```suggestion
        hi_def=True,
        loading=lazy
```

File: templates/automotive/index.html
Comment (minkyngkm) [reply templates/automotive/index.html]: I will just remove this column then!

**Review by petesfrench — APPROVED** [review]
LGTM :+1:

**Pull request merged**
